### PR TITLE
cmd/snap-bootstrap/initramfs-mounts: mount the snapd snap in run-mode too

### PR DESCRIPTION
**Pull request by anonymouse64**
To mount the snapd snap during run mode, we need to read the seed from the recovery system stored in the modeenv. This also means that we end up validating all of the seeds on every run-mode boot, but we will stop doing this when modeEnv is empty.

### CODE REVIEW

File: cmd/snap-bootstrap/cmd_initramfs_mounts.go [hunk 1]
```diff
@@ -298,7 +298,60 @@ func generateMountsModeRun() error {
...
+		isSnapdMounted, err := osutilIsMounted(filepath.Join(runMnt, "snapd"))
+		if err != nil {
+			return err
+		}
+
+		if !isSnapdMounted {
+			// load the recovery system  and generate mounts for kernel/base
+			systemSeed, err := seed.Open(seedDir, modeEnv.RecoverySystem)
+			if err != nil {
+				return fmt.Errorf("cannot open seed: %v", err)
```
Comment (pedronis) [inline cmd/snap-bootstrap/cmd_initramfs_mounts.go]: the other similar code doesn't rewrap errors, we should be consistent

File: cmd/snap-bootstrap/cmd_initramfs_mounts.go [hunk 2]
```diff
@@ -298,7 +298,60 @@ func generateMountsModeRun() error {
...
+			}
+			// load assertions into a temporary database
+			if err := systemSeed.LoadAssertions(nil, nil); err != nil {
+				return fmt.Errorf("cannot load assertions from seed %s: %v", modeEnv.RecoverySystem, err)
```
Comment (pedronis) [inline cmd/snap-bootstrap/cmd_initramfs_mounts.go]: same

File: cmd/snap-bootstrap/cmd_initramfs_mounts.go [hunk 3]
```diff
@@ -298,7 +298,60 @@ func generateMountsModeRun() error {
...
+			}
+			perf := timings.New(nil)
+			// TODO:UC20: LoadMeta will verify all the snaps in the
+			// seed, that is probably too much. We can expose more
+			// dedicated helpers for this later.
+			if err := systemSeed.LoadMeta(perf); err != nil {
+				return fmt.Errorf("cannot load metadata from seed %s: %v", modeEnv.RecoverySystem, err)
```
Comment (pedronis) [inline cmd/snap-bootstrap/cmd_initramfs_mounts.go]: same

Comment (anonymouse64) [reply cmd/snap-bootstrap/cmd_initramfs_mounts.go]: Undone, no longer wraps

File: cmd/snap-bootstrap/cmd_initramfs_mounts.go (hunk 2 above)
Comment (anonymouse64) [reply cmd/snap-bootstrap/cmd_initramfs_mounts.go]: Undone, no longer wraps

File: cmd/snap-bootstrap/cmd_initramfs_mounts.go (hunk 1 above)
Comment (anonymouse64) [reply cmd/snap-bootstrap/cmd_initramfs_mounts.go]: Undone, no longer wraps

**Pull request merged**